src/main/java/seedu/address/logic/commands/DeleteTaskCommand.java

```diff
@@ -71,22 +72,23 @@ public CommandResult execute(Model model) throws CommandException {
      *
      * @param student The student with the task to be deleted.
      */
-    public void deleteTaskFromStudent(Student student) {
+    public void deleteTaskFromStudent(Student student) throws CommandException {
         for (Task task : student.getTaskList()) {
             if (task.getTaskId().equals(targetTaskId)) {
                 logger.log(Level.INFO, "deleting task from student's taskList: " + task.getTaskId());
                 student.removeTask(task);
                 return;
             }
         }
+        throw new CommandException(String.format(MESSAGE_TASK_NOT_FOUND, targetTaskId));
     }
 
     /**
      * Deletes task from module's taskList.
      *
      * @param module The module with the task to be deleted.
      */
-    public void deleteTaskFromModule(Module module) {
+    public void deleteTaskFromModule(Module module) throws CommandException {
```

Comment:
Where is the CommandException thrown?

Reply:
Oops added this and forgot to remove it! Thanks for pointing out!